src/index.scss

```diff
@@ -109,29 +119,35 @@ code {
   top: 50%;
   z-index: 1;
   transform: translateY(-50%);
+  height: 100%;
```

Comment:
This is making the image look ugly on small screen sizes.
Instead of this just add `max-width:680px` This will stop the overflow of the image from both (x and y) directions.